client/package.json

```diff
@@ -10,7 +10,7 @@
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-scripts": "3.3.0",
-    "semantic-ui-react": "^0.88.2"
+    "semantic-ui-react": "latest"
```

Comment:
Probably shouldn't target `latest` version of a dependency, for consistency reasons.  You can put in the specific version you need